mmv1/products/spanner/api.yaml

```diff
@@ -102,8 +102,19 @@ objects:
         required: true
       - !ruby/object:Api::Type::Integer
         name: 'nodeCount'
-        description: 'The number of nodes allocated to this instance.'
-        default_value: 1
+        description: |
+          The number of nodes allocated to this instance. At most one of either node_count or processing_units can be present in terraforn. 
+        exactly_one_of:
```

Comment:
Could `conflicts` be applied here instead of `exactly_one_of`? That way we keep them mutually exclusive but neither are required